ms2query/create_sqlite_database.py

```diff
@@ -0,0 +1,407 @@
+    table_name:
+        Name of the table in the database that should store the tanimoto
+        scores. Default = tanimoto_scores.
+    col_name_identifier1:
+        Name of the first column of the table, this column will store numbers
+        that are the identifier of inchikeys. Default = 'identifier_1'
+    col_name_identifier2:
+        Name of the second column of the table, this column will store numbers
+        that are the identifier of inchikeys. Default = 'identifier_2'
+    col_name_score:
+        Name of the third column of the table, this column will store the
+        tanimoto scores. Default = 'tanimoto_score'
+    """
+    create_table_command = f""";
+    DROP TABLE IF EXISTS {table_name};
+    CREATE TABLE {table_name} (
+        {col_name_identifier1} INTEGER,
+        {col_name_identifier2} INTEGER,
+        {col_name_score} REAL,
+        PRIMARY KEY ({col_name_identifier1}, {col_name_identifier2})
+        );
+    """
+    conn = sqlite3.connect(sqlite_file_name)
+    cur = conn.cursor()
+    cur.executescript(create_table_command)
+    conn.commit()
+    conn.close()
+
+
+def add_tanimoto_scores_to_sqlite_table(sqlite_file_name: str,
+                                        npy_file_path: str,
+                                        col_name_identifier1: str =
+                                        'identifier_1',
+                                        col_name_identifier2: str =
+                                        'identifier_2',
+                                        col_name_score: str = 'tanimoto_score',
+                                        progress_bar: bool = True,
+                                        ):
+    """Adds tanimoto scores from npy file to sqlite table
+
+    The tanimoto scores are added to a sqlite file with in the first and
+    second column the identifiers that correspond to the index of the rows and
+    columns in the .npy file. The third row contains the tanimoto score. All
+    duplicates are removed, before adding to the sqlite table.
+
+    Args:
+    -------
+    sqlite_file_name:
+        Name of sqlite file to which the table should be added.
+    npy_file_path:
+        Path to the numpy file containing the tanimoto scores. The numpy file
+        is expected to contain a matrix filled with floats with nr of columns =
+        nr of rows.
+    col_name_identifier1:
+        Name of the first column of the table, this column will store numbers
+        that are the identifier of inchikeys. Default = 'identifier_1'
+    col_name_identifier2:
+        Name of the second column of the table, this column will store numbers
+        that are the identifier of inchikeys. Default = 'identifier_2'
+    col_name_score:
+        Name of the third column of the table, this column will store the
+        tanimoto scores. Default = 'tanimoto_score'
+    progress_bar:
+        Show progress bar if set to True. Default is True.
+    """
+    tanimoto_score_matrix = np.load(
```

Comment:
```suggestion
    # pylint: disable=too-many-arguments
    tanimoto_score_matrix = np.load(
```

Reply:
Currently pylint complains about having too many arguments (6) in this function. There are several ways to solve this, for instance by working with `**settings` or `**args` as part of the function input and then have a separate function to set the default values. But for now we could also skip this and just comment out the pylint complaint.